### ISSUE TITLE
False positive thrown on variable used in RelayQueryConfig

### ISSUE THREAD

**Issue by sgwilym** [apollographql/eslint-plugin-graphql#16]
``` js
export class NodeQueryConfig extends Relay.Route {
  static paramDefinitions = {
    nodeID: { required: true },
  };
  static routeName = 'NodeConfig';
  static queries = {
    node: () => Relay.QL`
      query {
        node(id: $nodeID)
      }
    `,
  };
}
```

Returns the error `Variable "$nodeID" is not defined`. Looks like the `paramDefinitions` property isn't being picked up.

**Comment by respectTheCode** [comment]
Maybe a good first step would be to allow us to opt out of the `NoUndefinedVariables` rule. This would at least allow Relay users to use the plugin until its fixed.

**Comment by helfer** [comment]
@respectTheCode I'd be open to merging a PR that turns this off for `Relay.QL`, if it's a simple change.

**Comment by stubailo** [comment]
Just need to add to this list and republish! https://github.com/apollostack/eslint-plugin-graphql/blob/921d15bcb3edae6219682bb6f1bd677dd03a9add/src/index.js#L38

**Comment by stubailo** [comment]
@helfer I added you as an owner so you can republish when necessary.

**Comment by jnwng** [comment]
looks like this was addressed (for now). please re-open if there's potentially a way to make sure we're taking into account `paramDefinitions` so we're not losing actual undefined variables
